docs/kms/IAM.md

```diff
@@ -8,6 +8,41 @@ onwards, MinIO will use the KMS provided keys to encrypt the IAM data instead
 of the cluster root credentials. If the KMS is not enabled, MinIO will store
 the IAM data as plain text in its backend.
 
+### Quick Start
+
+MinIO supports two (compatible) ways of encrypting IAM and configuration data.
```

Comment:
`compatible` might be read as "you can do both" vs "both of these work". I'm assuming that the envvar and KMS are mutually exclusive?

Reply:
Hm, yeah true... I meant that you can switch from `MINIO_KMS_SECRET_KEY`to KES (and in theory back as well). Will drop this.